src/Components/Admin/Trainers/Form/index.jsx

```diff
@@ -46,51 +42,49 @@ const AdminTrainerForm = () => {
     }
   };
 
-  const showSuccesModal = (data) => {
-    setModalMessage(data.message);
-    setModalTypeStyle('success');
-    setShowModal(true);
-    setShouldRedirect(true);
-  };
-  const showErrorModal = (error) => {
-    setModalMessage(error.message);
-    setModalTypeStyle('error');
-    setShowModal(true);
+  const showErrorToast = (message) => {
+    toast.error(message, {
+      duration: 2500,
+      position: 'top-right',
+      style: {
+        background: 'rgba(227, 23, 10, 0.5)'
+      },
+      iconTheme: {
+        primary: '#0f232e',
+        secondary: '#fff'
+      }
+    });
   };
 
   const createTrainer = async (data) => {
     try {
       const response = await dispatch(addTrainer(data));
-      showSuccesModal(response);
+      localStorage.setItem('toastMessage', response.message);
+      history.push('/admins/trainers');
     } catch (error) {
-      showErrorModal(error);
+      showErrorToast(error.message);
     }
   };
 
   const updateTrainerFunction = async (id, data) => {
     try {
-      const response = await dispatch(updateTrainer(id, data));
-      showSuccesModal(response);
+      const response = await dispatch(updateTrainer(data, id));
```

Comment:
Change the position of the params in the updateTrainer function like this: updateTrainer(id, data)